src/CubicSplines.jl

```diff
@@ -0,0 +1,72 @@
+module CubicSplines
+using LinearAlgebra
+
+#=
+This module is contained modified functions from DataInterpolations.jl (license below).
+We aren't depending on DataInterpolations because it's dependencies are very heavy.
```

Comment:
```suggestion
This module contains modified functions from DataInterpolations.jl (license below).
We aren't depending on DataInterpolations because its dependencies are very heavy.
```